services/order.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Any
+
+from django.db import transaction
+
+from db.models import Ticket, User, Order, MovieSession
+
+
+def create_order(tickets: list, username: str,
+                 date: datetime = None) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
Ensure that the `User` object retrieval handles the case where the user does not exist. You might want to catch the `User.DoesNotExist` exception to provide a more user-friendly error message or handle it appropriately.